closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -272,6 +281,11 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_loadCrypto
         ossl_ver = 1;
     }
 
+    if (trace) {
```

Comment:
These lines shouldn't be copied here, they need to be deleted.

Reply:
Actually it just seems the wrong string, it should be `"Supported OpenSSL version: %s\n"`

Reply:
Except on jdk17 there is already a trace message lower down on lines 432 and 435 which doesn't exist in the jdk11 code, so this one isn't needed.

Reply:
Deleted.